sdk/keyvault/azure_security_keyvault_keys/README.md

```diff
@@ -1,66 +1,370 @@
-# Azure KeyVault keys client library for Rust
+# Azure Key Vault keys client library for Rust
 
-Azure Key Vault is a cloud service that provides secure storage and management of sensitive information such as API keys, passwords, certificates, and cryptographic keys. The `azure_security_keyvault_keys` crate provides a client library for interacting with the Azure Key Vault Keys service.
+Azure Key Vault is a cloud service that provides secure storage of keys for encrypting your data. Multiple keys, and multiple versions of the same key, can be kept in the Azure Key Vault. Cryptographic keys in Azure Key Vault are represented as [JSON Web Key (JWK)](https://tools.ietf.org/html/rfc7517) objects.
+
+Azure Key Vault Managed HSM is a fully-managed, highly-available, single-tenant, standards-compliant cloud service that enables you to safeguard cryptographic keys for your cloud applications using FIPS 140-2 Level 3 validated HSMs.
+
+The Azure Key Vault keys library client supports RSA keys and Elliptic Curve (EC) keys, each with corresponding support in hardware security modules (HSM). It offers operations to create, retrieve, update, delete, purge, backup, restore, and list the keys and its versions.
+
+[Source code] | [Package (crates.io)] | [API reference documentation] | [Product documentation]
 
 ## Getting started
 
-### Install the crate
+### Install the package
 
-Install the Azure Key Vault keys library for Rust with cargo: 
+Install the Azure Key Vault keys client library for Rust with [Cargo]:
 
-```bash
+```sh
 cargo add azure_security_keyvault_keys
 ```
 
 ### Prerequisites
 
-- An [Azure subscription](https://azure.microsoft.com/free/)
-- A [Key Vault resource](https://docs.microsoft.com/azure/key-vault/quick-create-portal)
+* An [Azure subscription].
+* An existing Azure Key Vault. If you need to create an Azure Key Vault, you can use the Azure Portal or [Azure CLI].
+* Authorization to an existing Azure Key Vault using either [RBAC] (recommended) or [access control].
+
+If you use the Azure CLI, replace `<your-resource-group-name>` and `<your-key-vault-name>` with your own, unique names:
+
+```azurecli
+az keyvault create --resource-group <your-resource-group-name> --name <your-key-vault-name>
+```
+
+### Install dependencies
+
+Add the following crates to your project:
+
+```sh
+cargo add azure_identity azure_core tokio
```

Comment:
```suggestion
cargo add azure_identity tokio
```
I believe we don't need to call out core specifically unless we want to work with common types.